Rmarkdown/Multiple_Ascending_Dose_PD_continuous.Rmd

```diff
@@ -1,314 +1,311 @@
+
+When looking at summaries of continuous PD over time, there are several things to observe. Note the number of doses and number of time points or sampling schedule. Observe the overall shape of the average profiles. How does the profile change over time? Does the effect appear to increase and decrease quickly on a short time scale, or does is occur over a longer time scale? Is there clear separation between the profiles for different doses? Does the effect appear to increase with increasing dose? Do you detect a saturation of the effect? 
+
+### PD marker over time, colored by Dose, mean (95% CI) percentiles by nominal time
+
+```{r, cache = TRUE, fig.width = 8, fig.height = 4, warning = FALSE, message= FALSE} 
+
+#PD data
+gg <- ggplot(data = pd_data, 
+               aes(x=NOMTIME/24,y=LIDV, color = TRTACT_high2low, fill = TRTACT_high2low)) 
+gg <- gg + xgx_stat_ci(conf_level = .95)
+
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+gg <- gg + scale_x_continuous(breaks=seq(-1,9,1))
+gg <- gg + scale_y_continuous()
+gg <- gg + labs(y=pd_label)
+gg <- gg + xgx_annotate_status(status)
+
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width, height, dirs, "filename_main", status)
+print(gg)
+
+```
+
+### PD marker over time, faceted by Dose, mean (95% CI) percentiles by nominal time
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE}
+ 
+gg <- ggplot(data = pd_data %>% subset(DOSE>0), 
+               aes(x=NOMTIME/24,y=LIDV)) 
+gg <- gg + xgx_stat_ci(conf_level = .95)
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+gg <- gg + facet_grid(~TRTACT_low2high)
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+gg <- gg + scale_x_continuous(breaks=seq(-1,9,1))
+gg <- gg + scale_y_continuous()
+gg <- gg + labs(y=pd_label)
+
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width, height, dirs, "filename_main", status)
+print(gg)
+
+```
+
+## Explore variability
+
+Use spaghetti plots to visualize the extent of variability between individuals. The wider the spread of the profiles, the higher the between subject variability. Distinguish different doses by color, or separate into different panels. If coloring by dose, do the individuals in the different dose groups overlap across doses? Dose there seem to be more variability at higher or lower concentrations?
+
+
+### PD marker over time, colored by Dose, dots & lines grouped by individuals
+
+```{r, cache = TRUE, fig.width = 8, fig.height = 4, warning = FALSE, message= FALSE}
+ 
+ gg <- ggplot(data = pd_data %>% subset(DOSE>0), 

```

Comment:
don't need to filter out placebo, don't need to scale time

Reply:
removed scale time and re-enter PBO